launcher/game/project.rpy

```diff
@@ -435,6 +435,17 @@ init python in project:
 
             if self.projects_directory is not None:
                 self.scan_directory(self.projects_directory)
+                
+                # If a file called "projectPaths.txt" exists in the main project directory,
+                # also scan any directories listed there
+                extraPathsFName = os.path.join(self.projects_directory,"projectPaths.txt")
```

Comment:
I'd recommend using underscores instead of CamelCase to conform to the rest of project.